### PR TITLE
[ES] Add `P` missing stubs (1)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing P stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish, divided into 2 pull requests for better review.

Continued in: #10630

### CODE REVIEW

File: wiki/People/Mapping_Assistance_Team/MAT_Leaders/es.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - Mapping Assistance Team Leaders
+  - Líderes del Mapping Assistance Team
```
Comment (agatemosu) [inline wiki/People/Mapping_Assistance_Team/MAT_Leaders/es.md:5]:
```suggestion
  - líderes
```

File: wiki/People/Mapping_Assistance_Team/MAT_Leaders/es.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Líderes del MAT
+
+Los **líderes del MAT** supervisaron al [Mapping Assistance Team](/wiki/People/Mapping_Assistance_Team) (*MAT*). Eran responsables de gestionar y asesorar a los miembros del equipo, así como de crear conciencia sobre anuncios importantes y dar forma a las metas futuras y la dirección del equipo. Solo había un líder del MAT a la vez.
+
+## Historia
+
+::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) fue elegido como líder inaugural del MAT en la formación del grupo el 27 de diciembre de 2009.
+
+El cargo lo ocupó posteriormente ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) desde el 26 de agosto de 2010 hasta la disolución del MAT el 23 de Abril 2013.
```
Comment (agatemosu) [inline wiki/People/Mapping_Assistance_Team/MAT_Leaders/es.md:16]:
```suggestion
El cargo lo ocupó posteriormente ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) desde el 26 de agosto de 2010 hasta la disolución del MAT el 23 de abril 2013.
```

File: wiki/People/Mapping_Assistance_Team/es.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+tags:
+  - MAT
+---
+
+# Mapping Assistance Team
+
+El **Mapping Assistance Team** (***MAT***) era un grupo responsable de evaluar la calidad de los [beatmaps](/wiki/Beatmap) [pendientes](/wiki/Beatmap/Category#wip-and-pending) y recomendarlos para su [clasificación](/wiki/Beatmap/Category#ranked) en el sistema del [foro de modding](/wiki/Modding/Forum_modding). Sus recomendaciones se transmitían al [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*) colocando una [proto-burbuja](/wiki/Modding/Proto-bubble), y el BAT luego haría la decisión final sobre si el beatmap debía entrar en clasificación.
```
Comment (agatemosu) [inline wiki/People/Mapping_Assistance_Team/es.md:9]:
```suggestion
El **Mapping Assistance Team** (***MAT***) era un grupo responsable de evaluar la calidad de los [beatmaps](/wiki/Beatmap) [pendientes](/wiki/Beatmap/Category#wip-and-pending) y recomendarlos para su [clasificación](/wiki/Beatmap/Category#ranked) en el sistema de [modding en el foro](/wiki/Modding/Forum_modding). Sus recomendaciones se transmitían al [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*) colocando una [proto-burbuja](/wiki/Modding/Proto-bubble), y el BAT luego haría la decisión final sobre si el beatmap debía entrar en clasificación.
```

File: wiki/People/Beatmap_Appreciation_Team/es.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+stub: true
+tags:
+  - BAT
+  - Beatmap Appreciators
+  - Beatmap Approval Team
+  - apreciadores de beatmaps
```
Comment (agatemosu) [inline wiki/People/Beatmap_Appreciation_Team/es.md:7]:
```suggestion
  - apreciadores
```

**Comment by agatemosu** [comment]
I forgot to press send

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**